### DISCUSSION TITLE
Welcome, Sajan!

### DISCUSSION THREAD

**Discussion by Sajansharma0017** [ParanoidUser/yolo#1204] [Subscribe]
Leaving a message for the yolo badge, thanks.

**Comment by ParanoidUser** [comment]
Ahoy @Sajansharma0017, thank you for your interest in the project! To get a Yolo badge, simply follow the steps in https://github.com/ParanoidUser/yolo/discussions/18.

Note: Don't forget to assign a reviewer on step 4. 🔭

Another note: You should receive a repo collaborator invite that allows you to merge PRs (steps 5-6).

Let me know if you have any questions. Welcome aboard! 🚢⭐

**Comment by ParanoidUser** [comment]
@Sajansharma0017, if you've found this repository to be a valuable resource, go ahead and show some love with a stellar ⭐! Your support boosts its visibility within the community galaxy. 🔥🚀
